crates/router/tests/connectors/sample_auth.toml

```
@@ -17,4 +17,7 @@ key1 = "MyProcessingChannelId"
 api_key = "Bearer MyApiKey"
 
 [worldpay]
-api_key = "Bearer MyApiKey"
\ No newline at end of file
+api_key = "Bearer MyApiKey"
+[payu]
```

```suggestion

[payu]
```

Please include a blank line between TOML tables.